clase-6/tarea-2.js

```diff
@@ -45,58 +49,58 @@ const $buttonRemoveMember = document.getElementById("remove-member");
 
 $buttonRemoveMember.onclick = function (event) {
   let $dad = document.getElementById("container-annual-salary");
-  let $children = document.getElementById("label-members-annual-salary");
-  $dad.removeChild($children);
-  event.preventDefault();
-};
-
-function calculateMaxSalary(arraySalary) {
-  let maxSalary = arraySalary[0];
-  for (i = 0; i < arraySalary.length; i++) {
-    if (arraySalary[i] > maxSalary) {
-      maxSalary = arraySalary[i];
-    }
-  }
-  return maxSalary;
-};
-
-function calculateMinSalary(arraySalary) {
-  let minSalary = arraySalary[0];
-  for (i = 0; i < arraySalary.length; i++) {
-    if (arraySalary[i] < minSalary) {
-      minSalary = arraySalary[i];
-    }
-  }
-  return minSalary;
-};
+  $dad.lastChild.remove();
+  console.log(document.querySelectorAll(
+    "#container-annual-salary input"
+  ));
+  if (!document.querySelector("#label-members-annual-salary")) {
+    document.querySelector("#container-strong").className = "hidden";
+    document.querySelector("#button-calculate").className = "hidden";
 
-function calculateAverageAnnualSalary(arraySalary) {
-  let arraySalaryNoZero = [];
-  let averageAnnualSalary = 0;
-  let result = 0;
-  for (i = 0; i < arraySalary.length; i++) {
-    if (arraySalary[i] > 0) {
-      arraySalaryNoZero.push(arraySalary[i]);
-    }
-  }
-  for (let i = 0; i < arraySalaryNoZero.length; i++) {
-    result = arraySalaryNoZero[i] + result;
-  }
-  if (result > 0) {
-    averageAnnualSalary = result / arraySalaryNoZero.length;
   }
-  return averageAnnualSalary;
+  event.preventDefault();
 };
 
-function calculateAverageMensualSalary(averageAnnualSalary) {
-  const monthsInAYear = 12;
-  averageMensualSalary = averageAnnualSalary / monthsInAYear;
-  return averageMensualSalary;
+function getAnnualSalaryOfMember() {
+  
+  let arraySalary = [];
+  let inputAnnualSalary = inputNumber = document.querySelectorAll(
```

Comment:
esto no me cierra che, qué quisiste hacer acá?

Reply:
La verdad..... No sé jajaja.... lo voy a dejar así 

let inputNumber = document.querySelectorAll(
    "#container-annual-salary input"
  );
  console.log(document.querySelectorAll("#container-annual-salary input"));
  for (let i = 0; i < inputNumber.length; i++) {
    if (inputNumber[i].value != "") {
      arraySalary.push(Number(inputNumber[i].value));
    }
  }